src/components/ModalFlows/WrapTokenScreens/WrapTokenScreens.js

```diff
@@ -0,0 +1,103 @@
+import React, { useCallback, useMemo, useRef, useState } from 'react'
+import ModalFlowBase from '../ModalFlowBase'
+import WrapUnwrap from './WrapUnwrap'
+
+import { useAppState } from '../../../providers/AppState'
+import useActions from '../../../hooks/useActions'
+
+import BigNumber from '../../../lib/bigNumber'
+import { toDecimals } from '../../../utils/math-utils'
+
+const ZERO_BN = new BigNumber(toDecimals('0', 18))
```

Comment:
I think we can just do 0
```suggestion
const ZERO_BN = new BigNumber(0))
```